test/mocks/users.js

```diff
@@ -0,0 +1,68 @@
+const { UNPROCESSABLE_ENTITY, BAD_REQUEST } = require('../../app/errors');
+const {
+  CREATED,
+  VALIDATE_PASSWORD_MATCHES,
+  VALIDATE_EXISTS,
+  validationMessage
+} = require('../../config/constants');
+
+const passwordInvalid = '111av';
+
+const createUserInput = {
+  name: 'Julián',
+  lastName: 'Barrientos',
+  email: 'julian1.lopera@wolox.co',
+  password: 'password1'
+};
+
+const createUserOutput = {
+  data: {
+    name: createUserInput.name
+  },
+  message: CREATED
+};
+
+const uniqueEmailErrorMessage = 'email must be unique';
+
+const emailUniqueErrors = user => [
+  {
+    value: user.email,
+    message: uniqueEmailErrorMessage,
+    type: 'unique violation',
+    origin: 'DB',
+    path: 'email'
+  }
+];
+
+const unproccesableEmail = {
+  message: [uniqueEmailErrorMessage],
```

Comment:
Why this is an array?

Reply:
This message is an array when there are multiple constraints, for example:
```
{
    "message": [
        {
            "msg": "'name' is required",
            "param": "name",
            "location": "body"
        },
        {
            "msg": "'name' must be a string",
            "param": "name",
            "location": "body"
        },
        {
            "msg": "'name' must be a non-empty",
            "param": "name",
            "location": "body"
        }
    ],
    "internal_code": "bad_request"
}
```